test/server/graphql/v1/refundTransaction.test.js

```diff
@@ -267,6 +284,12 @@ describe('server/graphql/v1/refundTransaction', () => {
         where: { OrderId: transaction.OrderId },
       });
 
+      // Snapshot
+      await snapshotTransactionsForRefund(allTransactions);
+
+      // Collective balance should go back to 0
+      expect(await collective.getBalance()).to.eq(0);
```

Comment:
Before the fix above, this line would return `-250`: we were not accounting the refunded platform fees correctly